lib/scenario.js

```diff
@@ -1,9 +1,9 @@
-const promiseRetry = require('promise-retry');
-const event = require('./event');
-const recorder = require('./recorder');
-const assertThrown = require('./assert/throws');
-const { ucfirst, isAsyncFunction } = require('./utils');
-const parser = require('./parser');
+const promiseRetry = require("promise-retry");
+const event = require("./event");
+const recorder = require("./recorder");
+const assertThrown = require("./assert/throws");
+const { ucfirst, isAsyncFunction } = require("./utils");
```

Comment:
Yes sur ! Just it's could be fine if the linter will be fixed because when i run the lint command, i get an error due to eslint file.
But yes i will push with the good formatting

Reply:
I still have issue when commit 

```
> git -c user.useConfigOnly=true commit --quiet --allow-empty-message --file -

> codeceptjs@3.6.2 lint
> eslint bin/ examples/ lib/ test/ translations/ runok.js

✖ 4079 problems (4079 errors, 0 warnings)

husky - pre-commit hook exited with code 1 (error)
```